src/api/apiHooks/userIdentityHooks.ts

```diff
@@ -29,8 +29,15 @@ export const useRoleByUserId = (userId: string) => {
 
 export const useUpdateUser = (userId: string, user: ModalUserParams) => {
   return useUpdate<string, ModalUserParams, UserIdentity>(
-    `/identity/users`,
+    `/app/users`,
```

Comment:
Tại sao lại đổi từ identity -> app

Reply:
Dạ bởi vì ở dưới be build lại chức năng này nên đổi api ạ